pkg/controller/actions/deploy/action_deploy.go

```diff
@@ -120,9 +120,20 @@ func (a *Action) run(ctx context.Context, rr *odhTypes.ReconciliationRequest) er
 	controllerName := strings.ToLower(kind)
 
 	for i := range rr.Resources {
-		ok, err := a.deploy(ctx, rr, rr.Resources[i])
+		res := rr.Resources[i]
```

Comment:
nit:
reminded me a comment i saw yesterday:
` for _, r := range rr.Resources {`
		....

Reply:
this is a little bit different since in this case, you can't use the `r` as you shown because we pass the resource around and we should avoid aliasing.